src/application-base/ApplicationBase.jsx

```diff
@@ -51,10 +54,6 @@ const propTypes = {
    * The name of the theme to apply to the application using terra-theme-provider.
    */
   themeName: PropTypes.string,
-  /**
-   * If provided, the theme styles are applied to the entire document.
-   */
-  themeIsGlobal: PropTypes.bool,
```

Comment:
will want to add this prop removal in the changelog

Reply:
Yes